tools/continuous_integration.nim

```diff
@@ -0,0 +1,56 @@
+import os, osproc, strformat
+
+proc isAzureCI*(): bool =
+  getEnv("NIM_CI_Build_SourceBranchName").len > 0
```

Comment:
Please see how this is detected in testament.

Reply:
done, reused their code